rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/OracleTableMetaCache.java

```diff
@@ -100,7 +100,7 @@ private TableMeta resultSetMetaToSchema(DatabaseMetaData dbmd, String tableName)
         TableMeta tm = new TableMeta();
         tm.setTableName(tableName);
         String[] schemaTable = tableName.split("\\.");
-        String schemaName = schemaTable.length > 1 ? schemaTable[0] : dbmd.getUserName();
+        String schemaName = schemaTable.length > 1 ? schemaTable[0].toUpperCase() : dbmd.getUserName();
```

Comment:
If the table is created case sensitively, for example "test" converted to uppercase is wrong.

Reply:
You are right, the username of Oracle also case sensitive, but I think you can not toUpperCase directly(check out from here:https://docs.oracle.com/en/database/oracle/oracle-database/19/dbseg/managing-security-for-oracle-database-users.html#GUID-4723A344-402C-423C-A864-9D7975A02B49). 
You should consider this situation below
```
select * from "TEST".ACCOUNT_TBL;
select * from TEST.ACCOUNT_TBL;
select * from "Test".ACCOUNT_TBL;
```